Changes.md

```diff
@@ -1,6 +1,10 @@
 1.3.16.x (relative to 1.3.16.1)
 ========
 
+Fixes
+-----
+
+- File Browser : Windows only : Fixed bug in `HiddenFilePathFilter` that caused sequences to be treated as though they are hidden files.
```

Comment:
We should reference the relevant issue here with #.

Reply:
Right, thanks for the reminder. Added and squashed into https://github.com/GafferHQ/gaffer/pull/5831/commits/6108cc8d4f001d565a2623b0a88e63de061dcaf3